### DISCUSSION TITLE
Scalar expression to minimize please.

### DISCUSSION THREAD

**Discussion by Yikxinxin** [yalmip/YALMIP#1306] [Optimization modelling]
![image](https://github.com/yalmip/YALMIP/assets/104471890/2602d59d-cfc8-4538-a73b-b6273c69ff47)
Request support! I can't find a solution or a reason for encountering this problem.

**Comment by johanlofberg** [comment]
As the error message says, you've sent a non-scalar objective, which makes no sense
